IndicoV2.IntegrationTests/GraphQLRequest/GraphQLRequestTests.cs

```diff
@@ -34,8 +34,7 @@ public async Task RawQueryListDatasets_ShouldReturnResult()
             }";
             string operationName = "ListDatasets";
             dynamic variables = new { limit = 1 };
-            var result = await _graphQLRequestClient.Call(query, operationName, variables);
-
+            JObject result = await _graphQLRequestClient.Call(query, operationName, variables);

```

Comment:
coding convention would usually have this be `var`

Reply:
reverted